daemon/info_windows.go

```diff
@@ -11,6 +11,10 @@ import (
 
 // fillPlatformInfo fills the platform related info.
 func (daemon *Daemon) fillPlatformInfo(ctx context.Context, v *system.Info, sysInfo *sysinfo.SysInfo, cfg *configStore) error {
+	if _, ok := cfg.Features["windows-dns-proxy"]; ok {
+		v.Warnings = append(v.Warnings, `
+WARNING: Feature flag "windows-dns-proxy" has been removed, forwarding to external DNS resolvers is enabled.`)
+	}
```

Comment:
Thanks! Yes, I was thinking we'd need it only if someone has `false` set, but I guess this is good as well to tell them "remove this from your config"

We can remove this in v29.0 or so (maybe earlier, it's just a warning)